benchadapt/python/benchadapt/adapters/_adapter.py

```diff
@@ -190,7 +203,7 @@ def post_results(self) -> list:
             res_list.append(res)
 
         if error:
-            print(f"POST at {error_time} timed out twice:")
+            log.error(f"POST at {error_time} timed out twice:")
             raise error
```

Comment:
If there's value in this retry-on-timeout code, it should get migrated to benchclients so all requests get more robust handling, not just these particular [important!] ones. Made #824 for this